.rubocop.yml

```diff
@@ -42,3 +42,9 @@ Style/Documentation:
 Style/NumericPredicate:
   Exclude:
     - tasks/cops_documentation.rake
+
+Style/FormatStringToken:
```

Comment:
Took this right out of the rubocop .rubocop.yml. Turns out even rubocop doesn't like rubocop 😆 

https://github.com/rubocop/rubocop/blob/4fcf0ffb942b1d37e646e4230e7f1a8802feb9ca/.rubocop.yml#L34-L38